### PR TITLE
update installation instructions

### LINKED ISSUE #20: Dependency alert: kramdown version
GitHub found potential security vulnerabilities in the dependencies, see the message [here](https://github.com/eWaterCycle/ewatercycle.github.io/security/dependabot/Gemfile.lock/kramdown/open) and [Gemfile.lock](https://github.com/eWaterCycle/ewatercycle.github.io/blob/master/Gemfile.lock).

### PR DESCRIPTION
**Pull request by bvreede**
I created a new lockfile locally with `bundle install` from the current Gemfile, updating the versions of many dependencies.

Closes #20

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -183,68 +188,76 @@ GEM
...
       thread_safe (~> 0.1)
-    unicode-display_width (1.4.0)
+    unf (0.1.4)
+      unf_ext
+    unf_ext (0.0.7.7)
+    unicode-display_width (1.7.0)
+    zeitwerk (2.4.2)
 
 PLATFORMS
-  ruby
```
Comment (bvreede) [inline Gemfile.lock:244]: Given that I generated the lockfile automatically on a mac, could this throw a wrench?

**Comment by sverhoeven** [comment]
I would use Docker to run site locally 
```
rm Gemfile Gemfile.lock
docker run --rm --volume=$(pwd):/srv/jekyll -i -t  -p 127.0.0.1:4000:4000 jekyll/jekyll:pages jekyll serve
ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-linux-musl]
Configuration file: /srv/jekyll/_config.yml
            Source: /srv/jekyll
       Destination: /srv/jekyll/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
       Jekyll Feed: Generating feed for posts
                    done in 0.348 seconds.
 Auto-regeneration: enabled for '/srv/jekyll'
    Server address: http://0.0.0.0:4000
  Server running... press ctrl-c to stop.

```

**Comment by bvreede** [comment]
Thanks @sverhoeven — your docker instructions did not work on my machine and I'm not sure why, but I found [this post](https://martinpeck.com/jekyll/blog/containers/2019/02/02/build-and-deploy-jekyll-with-docker-part1/) and the first set did work on my end, so I used this. Would appreciate it if you could confirm these instructions to work!

Removing the lockfile does not hinder rendering of the site (either local or via docker) since it is regenerated from the Gemfile, and since this was the issue mentioned in the security alert I propose that we simply remove it altogether. Running the install will result in different lockfiles on different platforms anyway.
I suspect this does not actually resolve the security issue, since old versions of kramdown are still used, but since this is with the github-pages Gem list, I don't think we can fix it right now.

I did leave the original installation instructions, but removed the sudos (as per @Peter9192's [earlier comment](https://github.com/eWaterCycle/ewatercycle.github.io/pull/17#discussion_r596140113)).

File: README.md [hunk 2]
```diff
@@ -9,23 +9,40 @@ cycle.
...
 1. Clone the repository, move to its directory
     ```
     git clone git@github.com:eWaterCycle/ewatercycle.github.io.git
     cd ewatercycle.github.io
     ```
+1. Run with docker (recommended) or install Ruby and run locally
+
+## Install and run with Docker
+
+1. Ensure Docker is running.
```
Comment (Peter9192) [inline README.md]: I'm not sure I understand what this means.

File: README.md [hunk 3]
```diff
@@ -9,23 +9,40 @@ cycle.
...
+
+1. Run from the command line:
+```
+docker run --rm -it \
+  --volume="$PWD:/srv/jekyll" \
+  --volume="$PWD/vendor/bundle:/usr/local/bundle" \
+  -p 4000:4000 jekyll/jekyll:3.8 \
+  jekyll serve
```
Comment (Peter9192) [inline README.md]: This command doesn't work as expected for me: it builds and serves, but I cannot reach the address. Maybe this is due to my local settings, I'd have to delve into it to find out why.

File: README.md [hunk 4]
```diff
@@ -9,23 +9,40 @@ cycle.
...
     ```
-    sudo apt-get install ruby-full
+    apt-get install ruby-full
     ```
 1. Install the required gems
     ```
-    sudo gem install bundler
-    sudo bundle install
+    gem install bundler
+    bundle install
```
Comment (Peter9192) [inline README.md]:
```suggestion
    gem install bundler
    bundle config set --local path 'vendor/bundle'  # this avoids the need for sudo
    bundle install
```

File: README.md (hunk 3 above)
Comment (Peter9192) [reply README.md]: The difference with the local build is that this command serves at http://0.0.0.0:4000 whereas the local build serves at https://127.0.0.1:4000

File: README.md (hunk 2 above)
Comment (bvreede) [reply README.md]: On my OS, I needed to first turn on Docker before running a docker command from the command line. Does this work differently on Linux?
(And more to the point: what would be a better way to phrase this?)

Comment (Peter9192) [reply README.md]: So how do you turn it on, or confirm that it's running?

File: README.md (hunk 3 above)
Comment (bvreede) [reply README.md]: Thanks a lot for trying this out!

Could you try if this works:

```
docker run --rm -it \
  --volume="$PWD:/srv/jekyll" \
  --volume="$PWD/vendor/bundle:/usr/local/bundle" \
  -p 127.0.0.1:4000:4000 \
  jekyll/jekyll:3.8 \
  jekyll serve
```

If that doesn't work either, here is the docker command that @sverhoeven used:
```
docker run --rm -it \
  --volume=$(pwd):/srv/jekyll \
  -p 127.0.0.1:4000:4000 \
  jekyll/jekyll:pages \
  jekyll serve
```
does this work?

Comment (Peter9192) [reply README.md]: Unfortunately, they don't work either. Also the final server address is still 0.0.0.0:4000. Could be due to my local setup that this doesn't work, though..

**Review by Peter9192 — APPROVED** [review]
This works for me 👍

**Pull request merged**